src/routes/users.js

```diff
@@ -81,8 +81,8 @@ router.post('/reset_password_email', function (req, res, next) {
       host: 'smtp.mailtrap.io',
       port: 2525,
       auth: {
-        user: 'e1fa9b8d5ae44b',
-        pass: 'a199dbc739b3a2',
+        user: process.env.NODEMAILER_USER,
+        pass: process.env.NODEMAILER_PASSWORD
```

Comment:
Hi @shangong 

Thanks for reviewing! I couldnt replicate your error.  I generated the same query as yours with a 200 status
>Executing (default): SELECT "id", "firstName", "lastName", "email", "passwordHash", "contactNo", "resetPasswordToken", "resetPasswordExpiry", "createdAt", "updatedAt" FROM "Users" AS "User" WHERE "User"."email" = 'admin@techladies.co' LIMIT 1;
Executing (default): UPDATE "Users" SET "resetPasswordToken"=$1,"resetPasswordExpiry"=$2,"updatedAt"=$3 WHERE "id" = $4 RETURNING *
POST /users/reset_password_email 200 13454.624 ms - 62

On my postgresql, the token was generated with a expiry date of one day 
![image](https://user-images.githubusercontent.com/55618945/79630254-5cbf6480-8182-11ea-81f5-123116c96365.png)

I have made a commit to log the errors. If you dont mind, could you test it again and screenshot the error message?  Before that, also try to set up Mailtrap account here https://mailtrap.io/

Setting up Mailtrap

If Im not wrong, the error is likely because of the test account credentials . I am using Mailtrap  currently which is a mock account to send and receive emails. My account username and password are saved in .env file as NODEMAILER_USER and NODEMAILER_PASSWORD. To test the code, you will need to set up a free mailtrap account 

In the inbox page , click on the settings button 
![image](https://user-images.githubusercontent.com/55618945/79593014-6955a580-810d-11ea-9adf-837250a4c990.png)

You will be directed to a page that shows the username and password for your dummy email account. 

I initially placed the password and user directly, but thought its not a good idea to put it there even though its a dummy account so i place it at .env

When the mail is successfully sent out, you should check MailTrap inbox instead of your real email account for now.

Reply:
Excellent! It worked!